dask_jobqueue/jobqueue.yaml

```diff
@@ -3,12 +3,13 @@ jobqueue:
     name: dask-worker
 
     # Dask worker options
-    threads: 2
-    processes: 4
-    memory: 8GB
-    interface: null
-    death-timeout: 60
-    local-directory: null
+    cores: null                 # Total number of cores on a worker node
```

Comment:
Same remark for the _node_ in comment. Should'nt we use the term _job_?

Reply:
More importantly, I think we should provide a default cores number, especially if we fail if not given.

Reply:
What should the default cores number be?  Currently I'm somewhat inclined to fail and ask them to specify with a well worded error message.